### PR TITLE
support hyper getFsInfo()

### PR DESCRIPTION
**Pull request by heartlock**
Signed-off-by: heartlock 21521209@zju.edu.cn

### CODE REVIEW

File: manager/manager.go [hunk 1]
```diff
@@ -183,7 +190,7 @@ func New(memoryCache *memory.InMemoryCache, sysfs sysfs.SysFs, maxHousekeepingIn
 	// Register for new subcontainers.
 	eventsChannel := make(chan watcher.ContainerEvent, 16)
 
-	newManager := &manager{
+	newManager = &manager{
```
Comment (feiskyer) [inline manager/manager.go]: It's better move those definitions to L155 and only assign `fsInfo` and `eventsChannel` here.

**Comment by feiskyer** [comment]
Generally ok with only one nits. @resouer Do you have more comments?

**Comment by resouer** [comment]
I am ok with this

**Comment by feiskyer** [comment]
@heartlock Another `inHostNamespace` not defined error.

**Comment by feiskyer** [comment]
lgtm.

**Pull request merged**